parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -142,6 +142,8 @@ public static enum JobSummaryLevel {
   public static final String MAX_PADDING_BYTES    = "parquet.writer.max-padding";
   public static final String MIN_ROW_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.row.check.min";
   public static final String MAX_ROW_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.row.check.max";
+  public static final String MIN_ROW_COUNT_FOR_BLOCK_SIZE_CHECK = "parquet.block.size.row.check.min";
```

Comment:
Why fixing this on an old branch but not on the master?

Reply:
Because we use the 1.10 version,and I found that this issue has been fixed in [PARQUET-1920](https://issues.apache.org/jira/browse/PARQUET-1920).Thanks for your review and this pr looks like can be closed.